tsinfer/formats.py

```diff
@@ -1378,6 +1435,10 @@ def from_tree_sequence(
         with ``u`` sample nodes but no individuals will be translated into a SampleData
         file with ``u`` haploid individuals and ``u`` samples.
 
+        If metadata is present in the tree sequence it must either have a schema
```

Comment:
Should we say that metadata is transferred into the sample data file? It always is now, right (apart from node metadata)? Would it be worth saying that it's only top level, individuals, populations, and sites metadata that are taken from the TS? Presumably all others (e.g. node metadata) are ignored, and so could be any old schema.

So something like:

"Metadata associated with individuals, populations, sites, and at the top level of the tree sequence, is also stored in the appropriate places in the returned SampleData instance. Any such metadata must either have a schema defined[insert link to metadata schema discussion in tskit docs] or be JSON encodable text. Attempts to use a tree sequence where any such metadata is not JSON and has no schema will fail.

Reply:
Done.